src/cli/shared/Scope.option.ts

```diff
@@ -0,0 +1,30 @@
+/*
+* This program and the accompanying materials are made available under the terms of the
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at
+* https://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Copyright IBM Corp, 2019
+*
+*/
+
+import { ICommandOptionDefinition } from "@brightside/imperative";
+
+const MAX_LENGTH = 8;
+
+/**
+ * Imperative option for the "scope" parameter
+ *
+ */
+export const ScopeOption: ICommandOptionDefinition = {
+    name: "scope",
+    aliases: ["sc"],
+    type: "string",
+    stringLengthRange: [1, MAX_LENGTH],
+    impliesOneOf: ["csdgroup", "resgroup"],
+    description: "Specifies the name of the CICS System, or CICS System Group " +
+                 "(up to 8 characters) to target. " +
+                 "Use this parameter if you have not set the --cics-deploy-profile option."
```

Comment:
Could this override the value specified in the profile too?

Reply:
I started out preferring the idea that the user either uses a profile, or sets all the values, but not a mix of both; turns out that profiles don't work that way. I'm not sure if the value from the command line wins or loses, I'd like to think that it wins.